softvis3d-frontend/src/classes/Layout.ts

```diff
@@ -10,6 +10,10 @@ export default class Layout implements SelectOptionValue {
         this.description = description;
     }
 
+    public getValue(): string {
```

Comment:
I have standardised them now. The field is called `id` with all classes. This way the Interface-Method `getId()` is fitting.